uberon-base.owl

```diff
@@ -647703,7 +647703,7 @@
                 <owl:intersectionOf rdf:parseType="Collection">
                     <rdf:Description rdf:about="http://purl.obolibrary.org/obo/UBERON_0000178"/>
                     <owl:Restriction>
-                        <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/BFO_0000050"/>
+                        <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/RO_0001025"/>
```

Comment:
I don't see a PR where I made this exact change... [this seems related, but doesn't seem to re](https://github.com/obophenotype/uberon/pull/2379/files) seems related, but I don't see that exact edit.

Reply:
Thanks for checking. I guess I didn't put it together because of the skipped lines.... is there a way to view the lines above 647703 in this view?

![Screenshot 2022-04-21 at 12 52 05](https://user-images.githubusercontent.com/95643797/164511297-417bd7e7-1871-4fef-821e-94c22954f948.png)

Reply:
<img width="645" alt="Screenshot 2022-04-21 at 17 55 03" src="https://user-images.githubusercontent.com/76212760/164511808-c65b03e6-0f79-4997-b9a1-5ba9e462c536.png">

Reply:
Still can't expand the preceding lines in split view, at least on my screen. But glad you identified the source of the change.

Reply:
oh sorry misunderstood what you were asking, no don't think so, I just open the file in something like atom and view by line when I need to :)